ci-operator/step-registry/hypershift/dump/hypershift-dump-chain.yaml

```diff
@@ -5,6 +5,12 @@ chain:
     cli: latest
     commands: |-
       set -exuo pipefail
+
+      if [[ -f ${SHARED_DIR}/hive_kubeconfig ]]; then
+        echo "Using existing hive kubeconfig"
+        export KUBECONFIG=${SHARED_DIR}/hive_kubeconfig
+      fi
```

Comment:
`KUBECONFIG` is always set (it will point to the build cluster if the test does not put one in the shared directory), so the absence here should be an error (or an early exit, depending on what you want).